### ISSUE TITLE
Payment Journal posting

### ISSUE THREAD

**Issue by radovandokic** [Microsoft/AL#2878]
Hi,

When we post Payment Journal lines, which were exported to EFT file, "Check Ledger Entry"."Entry Status" remains Exported (never changed to Posted) and it is not possible to Void Check later. In previous versions we had status Transmitted, which was used (and still is used) for filtering in CU 12:

![image](https://user-images.githubusercontent.com/34686682/42633573-3522651a-85e1-11e8-9c5f-bb145d340163.png)

Basically, Check Ledger Entry is not updated during export of EFT file, and it looks like a bug.

Regards,
Radovan
